spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/CommitTimestamp.java

```diff
@@ -38,7 +38,7 @@
  * @since 1.2.2
  */
 public final class CommitTimestamp {
-	private static final Map<Object, Object> VALUES = new ConcurrentHashMap<>();
+	private final static Map<Object, Object> VALUES = new ConcurrentHashMap<>();
```

Comment:
This one too should still be `private static final`?

Reply:
Whoops - fixed.